### PR TITLE
V2v2 ui updates

### PR DESCRIPTION
**Pull request by lesleyjanenorton**
Template updates to bring us into alignment with Ryan's new and extra awesome treatment for Monitor.

### CODE REVIEW

File: views/partials/breach-card.hbs [hunk 1]
```diff
@@ -1,22 +1,32 @@
...
+        {{> breach-cards/data-classes }}
       {{/if}}
     </div>
     <div class="breach-card-link-wrap">
-      <span class="more-about-this-breach">{{ String.MoreInfoLink }}</span>
+      {{#if latestBreach}}
+        <span class="more-about-this-breach">{{ String.MoreInfoLink }}</span>
+      {{else}}
+        <span class="more-about-this-breach">{{ String.MoreInfoLink }}</span>
+      {{/if}}
```
Comment (pdehaan) [inline views/partials/breach-card.hbs]: The `if..else` looks the same here.

File: views/monitor.hbs [hunk 2]
```diff
@@ -1,35 +1,62 @@
...
+    {{#if featuredBreach.IsSensitive}}
+    <div class="sb-top-container">
+      <div class="row no-vertical-padding sb-top-wrapper">
+        <div class="sb-top"><!--top of sign up call out--></div>
+      </div>
+    </div>
+     {{/if}}
+  {{else}}
+    <div class="latest-breach-headline-wrapper">
+      <h3 class="latest-breach-headline">{{{ getString "latest-breach" brandName="<span class='fxm'>Firefox Monitor</span>" }}}</h3>
```
Comment (pdehaan) [inline views/monitor.hbs]: Not sure how localizing the `brandName` would work here.
I tried doing a <kbd>git grep "\-brand-name = "</kbd>, but it looks like `-brand-name` is just "Firefox" in most locales:

```sh
locales/en/app.ftl:-brand-name = Firefox
```

vs `-product-name` which is usually "Firefox Monitor":

https://github.com/mozilla/blurts-server/blob/db442bf6b24582517412f416fa5a1d54eca1e80f/locales/en/app.ftl#L6

... with a couple exceptions:

```sh
git grep "\-product-name = " | grep -v " = Firefox Monitor"

locales/km/app.ftl:-product-name = កម្មវិធី​ត្រួតពិនិត្យ Firefox
locales/km/email-strings.ftl:-product-name = កម្មវិធី​ត្រួតពិនិត្យ Firefox
locales/ml/app.ftl:-product-name = ഫയർഫോക്സ് മോണിറ്റർ
locales/ml/email-strings.ftl:-product-name = ഫയർഫോക്സ് മോണിറ്റർ
```

Not sure if it's worth renaming the arg to `productName` for consistency, but that's super minor nit.

File: template-helpers/dashboard.js [hunk 3]
```diff
@@ -6,19 +6,27 @@ const { makeBreachCards } = require("./breaches");
 function getBreachesForEachEmail(args) {
   const verifiedEmails = args.data.root.verifiedEmails;
   const locales = args.data.root.req.supportedLocales;
+  let breachesFound = false;
 
   verifiedEmails.forEach(email => {
     const breachCards = makeBreachCards(email.breaches, locales);
     email.foundBreaches = {};
     email.foundBreaches.firstFourBreaches = breachCards.slice(0, 4);
     email.foundBreaches.remainingBreaches = breachCards.slice(5, breachCards.length);
```
Comment (pdehaan) [inline template-helpers/dashboard.js]: Unrelated, but I think this was causing a bug:
https://github.com/mozilla/blurts-server/issues/902#issuecomment-492004302

Not sure if this should be `.slice(4, breachCards.length);`

File: views/scan.hbs [hunk 4]
```diff
@@ -1,35 +1,40 @@
-<main class="container bg-dark clear-header" {{> analytics/results_page_label }}>
+<main class="container bg-light clear-header" {{> analytics/results_page_label }}>
   <div class="row jst-cntr scan-results">
 
-    <div class="col-9 headline-col no-vertical-padding">
+    <div class="col-9 headline-col no-vertical-padding">{{getArgs}}
```
Comment (groovecoder) [inline views/scan.hbs]: Why is this call to `getArgs` here? Debugging?

File: template-helpers/sign-up-banners.js [hunk 5]
```diff
@@ -37,30 +37,30 @@ function monitorFeaturesList(args) {
...
+      subtitleStringId: "keep-your-data-safe",
       pictogramPath: "advice",
     },
   ];
 
   features.forEach(feature => {
-    feature.title = LocaleUtils.fluentFormat(locales, feature.titleId);
-    feature.subtitle = LocaleUtils.fluentFormat(locales, feature.subtitleId);
+    feature.title = LocaleUtils.fluentFormat(locales, feature.titleStringId);
+    feature.subtitle = LocaleUtils.fluentFormat(locales, feature.subtitleStringId);
```
[the remark this answers is not in the document]
Comment (lesleyjanenorton) [reply template-helpers/sign-up-banners.js]: Not really - it just makes it a bit easier in the event of a later string change.

File: template-helpers/breach-detail.js [hunk 6]
```diff
@@ -225,7 +224,7 @@ function getBreachDetail(args) {
 }
 
 
-function soupedUpDataClasses(locales, breach) {
+function soupedUpDataClasses(locales, breach, forBreachCard = false) {
```
[the remark this answers is not in the document]
Comment (lesleyjanenorton) [reply template-helpers/breach-detail.js]: haha yes

**Pull request merged**